### ISSUE TITLE
[BUG]ts_dataframe

### ISSUE THREAD

**Issue by amir78pgd** [autogluon/autogluon#2655]
- [x] I have checked that this bug exists on the latest stable version of AutoGluon
- [x] and/or I have checked that this bug exists on the latest mainline of AutoGluon via source installation

**Describe the bug**

pandas dataframe

	Area.x	Date			Vehicle Demand
0	1	2021-01-01 00:07:00	20
1	1	2021-01-01 01:07:00	26
2	1	2021-01-01 02:07:00	20
3	1	2021-01-01 03:07:00	6
4	1	2021-01-01 04:07:00	12
...	...	...	...
118603	1	2021-01-07 19:07:00	6
118604	1	2021-01-07 20:07:00	3
118605	1	2021-01-07 21:07:00	5
118606	1	2021-01-07 22:07:00	5
118607	1	2021-01-07 23:07:00	3
118608 rows × 3 columns

ts_dataframe for Area.x == 1

                        Vehicle Demand
timestamp	
2021-01-01 00:07:00	20
2021-01-01 00:07:00	20
2021-01-01 00:07:00	20
2021-01-01 00:07:00	20
2021-01-01 00:07:00	20
...	...
2021-01-07 23:07:00	3
2021-01-07 23:07:00	3
2021-01-07 23:07:00	3
2021-01-07 23:07:00	3
2021-01-07 23:07:00	3
118608 rows × 1 columns

**Expected behavior**

ts_dataframe for Area.x == 1

                        Vehicle Demand
timestamp	
2021-01-01 00:07:00	20
2021-01-01 01:07:00	26
2021-01-01 02:07:00	20
2021-01-01 03:07:00	6
2021-01-01 04:07:00	12
...	...
2021-01-07 19:07:00	6
2021-01-07 20:07:00	3
2021-01-07 21:07:00	5
2021-01-07 22:07:00	5
2021-01-07 23:07:00	3

**To Reproduce**
A minimal script to reproduce the issue. Links to Colab notebooks or similar tools are encouraged.  
If the code is too long, feel free to put it in a public gist and link it in the issue: https://gist.github.com.  
In short, we are going to copy-paste your code to run it and we expect to get the same result as you.

**Screenshots**

**Installed Versions**
Which version of AutoGluon are you are using?  
If you are using 0.4.0 and newer, please run the following code snippet:
<details>

```python
# Replace this code with the output of the following:
from autogluon.core.utils import show_versions
show_versions()
```

</details>

**Additional context**

**Comment by amir78pgd** [comment]
[temp_2.xlsx](https://github.com/autogluon/autogluon/files/10361781/temp_2.xlsx)
the attached file had other area codes also like 2, 3, 4 etc but I removed them to reduce the file size

**Comment by amir78pgd** [comment]
this is not a bug, I had made a mistake in my code.
